projects/Mallard/src/hooks/use-issue-summary.tsx

```diff
@@ -43,10 +48,17 @@ const IssueSummaryProvider = ({ children }: { children: React.ReactNode }) => {
         null,
     )
     const [error, setError] = useState<string>('')
-    const hasConnected = useRef(false)
+    const isConnected = useRef<boolean>(true) // assume we are connected to start with
+    const { data } = useQuery(
+        gql`
+            {
+                maxAvailableEditions @client
+            }
+        `,
+    )
```

Comment:
you can shorten that if you wish on a single line by using `gql` as normal function:

    gql('{ maxAvailableEditions @client }') 

which won't cause reformating on several lines.

Reply:
Don't use the base version of `useQuery`, at least not without checking for errors. Use our own version in `hooks/apollo.ts`

Reply:
Will talk IRL about this ... there are some usability things about the wrapper that make it a bit more verbose to use, might be worth a tiny refactor.

Reply:
Sure, let's make sure we're all happy about it. Maybe the version that was returning `null` when loading wasn't too bad after all :D  it's was a little less boilerplate

Reply:
Regardless your code is not fully correct because it'll ignore errors. Please rethrow errors returned by `useQuery` or otherwise handle them.